### PR TITLE
Plymouth website project story

### PR DESCRIPTION
**Pull request by mybits**

### CODE REVIEW

File: source/project-stories/plymouth-university.erb [hunk 1]
```diff
@@ -0,0 +1,328 @@
...
+
+introduction:
+  title: "New website led by Unboxed"
+  content: "With the arrival of expensive tuition fees, universities have had to become more commercially oriented. Plymouth University realised that prospective students learn about them primarily through their website, and that the current site did not reflect their quality. A team from Unboxed worked alongside developers from the University to rapidly overhaul the website, adding functionality in two-week sprints."
+  image: "/project-stories/plymouth-university/plymouth-website.jpg"
+  image_description: "Plymouth website screenshot"
+
+plymouth_cio_video:
+  youtube_id: "nTSKD1PG1e4"
+  thumbnail: "/project-stories/plymouth-university/video-still.jpg"
```
Comment (tomsabin) [inline source/project-stories/plymouth-university.erb:18]: Getting a 404 when requesting this image, though I may fix this myself so I can update the style to be the same as SH:24 if that's okay with you.

File: source/project-stories/plymouth-university.erb [hunk 2]
```diff
@@ -0,0 +1,328 @@
...
+
+team_quote:
+  quote: "Unboxed Consulting excelled in that they were able to put a team down in Plymouth to skill up our in-house team, something that many suppliers are unwilling to do."
+  source: "Paul Westmore, Plymouth University"
+site:
+  link: "https://www.plymouth.ac.uk/"
+  text: "Visit the site and check it out for yourself"
+
+team:
+  people:
```
Comment (tomsabin) [inline source/project-stories/plymouth-university.erb:60]: Do we need this double nesting? Seems we are missing the description from here

Comment (tomsabin) [inline source/project-stories/plymouth-university.erb:60]: Fixed in https://github.com/unboxed/ubxd_web_refresh/commit/9563f01ab0f25c5a58d625b30c61961b1188af23

File: source/project-stories/plymouth-university.erb (hunk 1 above)
Comment (tomsabin) [inline source/project-stories/plymouth-university.erb:18]: Have updated the images in https://github.com/unboxed/ubxd_web_refresh/commit/3e3eced087c945fd738dced2ee00f305701bcf5b. It was referencing the .jpg files but only .png files were present.

Comment (mybits) [inline source/project-stories/plymouth-university.erb:18]: Oh cool, thanks a lot!

**Comment by tomsabin** [comment]
LGTM :last_quarter_moon:

**Pull request merged**